### PR TITLE
Add basic support for yambar themes

### PR DESCRIPTION
**Pull request by dimkr**
I need to add some code that selects the battery and network interface to use instead of hardcoding BAT0 and wlp3s0, but that can wait for another PR.

~~(And something is wrong with my implementation of wlr-foreign-toplevel-management, yambar ignores some windows and sometimes displays the wrong title)~~ should be OK now

(screenshots from bookworm64 built from this PR plus #3155)

With a virtual sound card and the right network interface:

tray

buntoo

### CODE REVIEW

File: woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    true: {string: {text: 🔇}}
+                    false:
+                      ramp:
+                        tag: volume
+                        items:
+                          - string: {text: "🔈{percent}%"}
+                          - string: {text: "🔉{percent}%"}
+                          - string: {text: "🔊{percent}%"}
+    - network:
+        name: wlp3s0
```
Comment (01micko) [inline woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in]: need to figure out how to un-hardcode this.

Comment (dimkr) [reply woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in]: I think this "widget" need to be replaced with a script that runs `connmanctl monitor` and outputs 🌐 or 🚫 every time internet connection is restored or lost on all interfaces. yambar supports this kind of notifications from a shell script.

Comment (dimkr) [reply woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in]: Done, see c7c45c8

Comment (dimkr) [reply woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in]: disabled
ready

**Review by 01micko — APPROVED** [review]
`conman` specific, but that doesn't really matter. I may go that way sooner or later

**Comment by dimkr** [comment]
> `conman` specific, but that doesn't really matter. I may go that way sooner or later

True, but if it's important, I can add fallback to an internal implementation based on netmon_wce.

**Comment by 01micko** [comment]
> > `conman` specific, but that doesn't really matter. I may go that way sooner or later
> 
> True, but if it's important, I can add fallback to an internal implementation based on netmon_wce.

In the context of 'bleeding edge', not important.

**Comment by dimkr** [comment]
Merged into #3155.

**Pull request closed without merging**